stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -224,7 +245,7 @@ class FPDF:
     def write(self, h: Any | None = ..., txt: str = ..., link: str = ...) -> None: ...
     def image(
         self,
-        name,
+        name: str | Image.Image | BytesIO | StrPath,
```

Comment:
In the `else` branch, it calls `str(name)`, so in theory all objects are valid here, but I feel that using `StrPath` is safer.